### DISCUSSION TITLE
Python not installed

### DISCUSSION THREAD

**Discussion by bbuster2** [ej52/proxmox-scripts#158] [Q&A]
Trying to install Nginx Proxy Manager on Proxmox 8.1.3.
Using this command:

`sh -c "$(wget --no-cache -qO- https://raw.githubusercontent.com/ej52/proxmox/main/install.sh)" -s --app nginx-proxy-manager`

Getting the following error:

```
    _   __      _               ____                           __  ___                                 
   / | / /___ _(_)___  _  __   / __ \_________ __  ____  __   /  |/  /___ _____  ____ _____ ____  _____
  /  |/ / __  / / __ \| |/_/  / /_/ / ___/ __ \| |/_/ / / /  / /|_/ / __  / __ \/ __  / __  / _ \/ ___/
 / /|  / /_/ / / / / />  <   / ____/ /  / /_/ />  </ /_/ /  / /  / / /_/ / / / / /_/ / /_/ /  __/ /    
/_/ |_/\__, /_/_/ /_/_/|_|  /_/   /_/   \____/_/|_|\__, /  /_/  /_/\__,_/_/ /_/\__,_/\__, /\___/_/     
      /____/                                      /____/                            /____/             
 
✔ Operating System Updated 
✔ Dependencies Installed 
✔ Rust v1.74.0 Installed 
✘ Python not Installed
E: Sub-process /usr/share/proxmox-ve/pve-apt-hook returned an error code (1)
E: Failure running script /usr/share/proxmox-ve/pve-apt-hook

[ERROR] on line 99 with exit code 100 while executing command pkg_del
root@proxmox:~#
```

Do i need to install python manually?

**Comment by 3bsalam-1** [comment]
yes , if you use Debian based linux run this command in terminal: apt install python3
or if you use windows just install it from python's official website

**Reply by CABESTAN-NET** [reply]
Hello

I have the same problem

✘ Python not Installed
E: Sub-process /usr/share/proxmox-ve/pve-apt-hook returned an error code (1)
E: Failure running script /usr/share/proxmox-ve/pve-apt-hook

[ERROR] on line 99 with exit code 100 while executing command pkg_del

And when I do : 
 # apt install python3
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
python3 is already the newest version (3.11.2-1+b1).
0 upgraded, 0 newly installed, 0 to remove and 11 not upgraded.

How I can do ???

thaks
